docs/src/components/DocsLayout.tsx

```diff
@@ -59,7 +67,8 @@ class DocsLayout extends React.Component<any, any> {
     // Anchor links has issues with <base>
     // https://stackoverflow.com/questions/8108836/make-anchor-links-refer-to-the-current-page-when-using-base
     document.querySelectorAll('a.anchor-link').forEach(link => {
-      link.setAttribute('href', `${document.location.href}${link.getAttribute('href')}`)
+      const value = `${this.getUrlWithoutHash(document.location.href)}${link.getAttribute('href')}`
```

Comment:
Can you please explain why `getUrlWithoutHash()` is needed there?

Reply:
Because document.location.href can already end with a hash, so we would have two hashes in a link, which is wrong

Reply:
So we can use `location.pathname` instead? Will it work?

Reply:
Use `origin`, `pathname` instead